apps/to-do-api/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "to-do-api",
+  "version": "0.0.0",
+  "type": "module",
+  "description": "Azure Function REST API for To Do List",
+  "license": "MIT",
+  "main": "./dist/main.js",
+  "scripts": {
+    "clean": "shx rm -rf ./dist ./src/generated",
+    "build": "tsc",
+    "typecheck": "npm run build -- --noemit",
```

Comment:
yarn?

Reply:
I used `npm` only to execute scripts, but I'll fix it with `yarn`